src/xrpld/core/detail/DatabaseCon.cpp

```diff
@@ -276,16 +268,6 @@ setup_DatabaseCon(Config const& c, std::optional<beast::Journal> j)
     setPragma(setup.txPragma[1], "journal_size_limit", journal_size_limit);
     setPragma(setup.txPragma[2], "max_page_count", max_page_count);
 
-#if (ULONG_MAX > UINT_MAX) && !defined(NO_SQLITE_MMAP)
-    setup.txPragma[3] = "PRAGMA mmap_size=17179869184;";
```

Comment:
sorry if I was unclear; I meant in https://github.com/XRPLF/rippled/pull/5135#discussion_r1764989451 to change this as-if the `#if` condition was always true, which means to keep this line and remove the directive `#if` above and the block '#else .... #endif` below. This means that txPragma will always have size 4, not 3